app.js

```diff
@@ -33,7 +33,17 @@ app.use(serveFavicon(path.join(__dirname + '/public/favicon.ico')));
 app.use(bodyParser.json());
 app.use(serveStatic(path.join(__dirname, 'public')));
 
+const defaultReferrers = [
+	"http://stylifyme.com",
+	"http://www.stylifyme.com",
+	"http://stylify.herokuapp.com",
+	"http://localhost:9185",
+	"http://localhost:7210"
+];
 
+const validReferrers = (process.env.VALID_REFERRERS || defaultReferrers.join(",")).split(",").map(i => i.trim());
```

Comment:
Haven't tested, but I think something like this is a little cleaner.

```suggestion
const validReferrers = process.env.VALID_REFERRERS ? process.env.VALID_REFERRERS.split(",").map(i => i.trim()) : defaultReferrers;
```

Reply:
Roger that